src/server/views/dashboard/lists-edit.njk

```diff
@@ -1,141 +1,114 @@
 {% extends "./dashboard.njk" %}
 
+{% block aside %}
+  <div class="back-links__item">
+    <a class="govuk-link govuk-back-link" href="/dashboard/lists">Back</a>
+  </div>
+  <div class="back-links__item">
+    <a class="govuk-link govuk-back-link" href="/dashboard/lists">Back</a>
+  </div>
+{% endblock %}
+
+
 {% block dashboard %}
-  <h1 class="govuk-heading-l">
-    {% if listId === 'new' %}
-      Create new list
-    {% else %}
-      Editing list {{ listId }}
+  <div>
+    {% if publisher.change %}
```

Comment:
remove. this shouldn't be around anymore, I have changed to flashed message via messages.successBannerMessage